### DISCUSSION TITLE
How to show all pages on main page for folder?

### DISCUSSION THREAD

**Discussion by mrvectopus** [shuding/nextra#2883] [Q&A]
I mean something like a list of all the pages in a Blog theme, how can I do the same in a Docs theme?
https://frontendweb3.github.io/nextra-blog/posts

There is a Programs folder and main page "programs.mdx" for this folder, it contains windows.mdx, macos.mdx, linux.mdx pages.

How to show list with title and url of contains pages (windows.mdx, macos.mdx, linux.mdx) on main page (programs.mdx)?

UPD: I found solution https://github.com/shuding/nextra/discussions/2528

**Comment by Srinath10X** [comment]
Install the docs theme, then add the blog folder and make /blog
